### PR TITLE
Fixed the documentation for Vector.angleBetween to indicate that it respects angleMode.

### PR DESCRIPTION
**Pull request by sflanker**
Changes:

Modifies the documentation comments for p5.Vector.prototype.angleBetween to indicate that it respects angleMode and will return the angle in either radians or degrees depending on the current value.

#### PR Checklist

Documentation only change.

### CODE REVIEW

File: src/math/p5.Vector.js [hunk 1]
```diff
@@ -1549,7 +1549,10 @@ p5.Vector.prototype.rotate = function rotate(a) {
 };
 
 /**
- * Calculates and returns the angle (in radians) between two vectors.
+ * Calculates and returns the angle between two vectors. This function will take
+ * the current <a = src="#/p5/angleMode">angleMode</a> into consideration, and
```
Comment (outofambit) [inline src/math/p5.Vector.js:1553]: thanks @sflanker! this looks good, but we need to use `href` for the link to work
```suggestion
 * the current <a href="#/p5/angleMode">angleMode</a> into consideration, and
```

Comment (sflanker) [reply src/math/p5.Vector.js:1553]: Oops, I copied this from the comments on the `heading` function without paying enough attention.

**Comment by lmccart** [comment]
thanks @sflanker!

File: src/math/p5.Vector.js (hunk 1 above)
Comment (outofambit) [reply src/math/p5.Vector.js]: no problem! thank you!

**Pull request merged**